virttest/utils_sriov.py

```diff
@@ -99,6 +99,23 @@ def get_pf_pci(session=None):
             return pci_info.get('pci_id')
 
 
+def get_pf_info_by_pci(pci_id, session=None):
+    """
+    Get the pci info by the given pci id
+
+    :param pci_id: PF's pci id, eg. 0000:3b:00.0
+    :param session: The session object to the host
+    :return: Dict, pf's info,
+        eg. {'driver': 'ixgbe', 'pci_id': '0000:3b:00.0',
+             'iface': 'ens1f0', 'status': 'up'}
+    """
+    pf_info = get_pf_info(session=session)
```

Comment:
how about the get_pf_info() return None? may need add an judgement here.

Reply:
If there is no pfs, it'll return '{}' and the following for loop will do nothing.